pkg/controllers/deprovisioning/expiration_test.go

```diff
@@ -188,23 +179,18 @@ var _ = Describe("Expiration", func() {
 		// deprovisioning won't delete the old node until the new node is ready
 		var wg sync.WaitGroup
 		ExpectTriggerVerifyAction(&wg)
-		ExpectMakeNewMachinesReady(ctx, env.Client, &wg, cluster, cloudProvider, 1)
+		ExpectMakeNewNodesReady(ctx, env.Client, &wg, 1)
 		ExpectReconcileSucceeded(ctx, deprovisioningController, types.NamespacedName{})
 		wg.Wait()
 
-		// Cascade any deletion of the machine to the node
-		ExpectMachinesCascadeDeletion(ctx, env.Client, machine)
-
 		// Expect that the new machine was created, and it's different than the original
-		ExpectNotFound(ctx, env.Client, machine, node)
-		machines := ExpectMachines(ctx, env.Client)
+		ExpectNotFound(ctx, env.Client, node)
 		nodes := ExpectNodes(ctx, env.Client)
-		Expect(machines).To(HaveLen(1))
 		Expect(nodes).To(HaveLen(1))
-		Expect(machines[0].Name).ToNot(Equal(machine.Name))
 		Expect(nodes[0].Name).ToNot(Equal(node.Name))
 	})
-	It("should uncordon nodes when expiration replacement partially fails", func() {
+	It("should uncordon nodes when expiration replacement fails", func() {
```

Comment:
This is effectively the same test, since we are validating the ability of the test to uncordon after we decide to launch. Whether it's 3 or 1 I don't think makes much of a difference here